.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "rules": {
+    "multiline-ternary": 0,
```

Comment:
what does this one do?

Reply:
No idea, I just copied them from the package json haha.  I'm guessing it stops eslint complaining about multiline ternaries.  Linting discussion is for another day but you can see the sort of config I like using in the test-dapp repo.